src/io_uring/src/operation/mod.rs

```diff
@@ -88,6 +112,20 @@ impl<T> Operation<T> {
         self.addr
     }
 
+    // Needed for proptesting.
+    #[cfg(test)]
+    pub fn ordered(&mut self) {
+        self.flags |= 1 << bindings::IOSQE_IO_LINK_BIT;
+    }
+
+    // Needed for proptesting.
+    #[cfg(test)]
+    pub fn offset(&self) -> Option<u64> {
+        self.offset
+    }
+
+    #[cfg(test)]
+    // Needed for proptesting.
```

Comment:
If we test them, maybe we could also expose these methods in the public interface.